Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
> I thought default is the sanest option to make pugixml work anywhere (without any -std= option)

So generally I'm the only person using the Makefile :) so the default options are set up for convenience of local testing - -std=c++11 being the choice since it activates parts of pugixml code and tests that are otherwise inaccessible.

Not sure why you'd use the Makefile in other cases - it's meant as a tool to work with pugixml, not to build pugixml, since to build pugixml you could just add the .cpp to your build.